sql/testdata/subquery

```diff
@@ -135,3 +135,8 @@ query IT
 SELECT * FROM kv WHERE k = (SELECT k FROM kv WHERE (k, v) = (1, 'one'))
 ----
 1 one
+
+query B
```

Comment:
You can't merge this if the test is failing. I'd change this to expect the error that is occurring and add a `TODO` comment to fix.